src/pages/consent.mdx

```diff
@@ -0,0 +1,114 @@
+
+- Allow certain addresses to message you
+- Block addresses you don't want to hear from
+
+**For group chats**, it's about which groups you want to participate in:
+
+- Join the group to receive its messages
+- Block to stop receiving messages from the group
+
+In both cases, new messages and invites stay in your message requests until you make a decision.
+
+## Privacy-First Design
+
+Your choices about who can message you are private. This means:
+
+- Only you can see who you've allowed or blocked
+- Your choices are encrypted and visible only to you
+- No one else can see your messaging preferences
+- Blocked users don't know they're blocked
+
+## How Consent Works
+
+The XMTP network maintains three private states for each potential messaging relationship:
+
+```rust
+pub enum ConsentState {
+    Unknown,    // Default state for new contacts
+    Allowed,    // Contacts you've allowed
+    Denied,     // Contacts you've blocked
+}
+```
+
+:::note Implementation Note
+Currently, consent preferences are client-side filters - they help apps determine which messages to show in your inbox, but don't prevent message transmission at the network level. The network plans to implement consent enforcement in the future.
+:::
+
+When you receive a new message, it appears in your message requests. Apps typically show:
+
+- Main inbox: Messages from allowed addresses and joined groups
+- Message requests: New messages and invites you haven't decided about
+- Hidden: Messages from blocked addresses and groups
+
+## The Future of Consent
+
+XMTP's consent system is expanding in two major ways:
+
+### 1. Programmable Consent
+
+Today, individual apps implement their own ways of handling consent granting - through app-specific invite links, group joining mechanisms, or eligibility rules. We are creating an interoperable standard that works across the entire ecosystem:
+
+- Apps can generate and validate consent links using a common protocol
+- Links work consistently across all XMTP apps
+- Supports flexible eligibility criteria (token ownership, DAO membership, etc.)
+- Enables ecosystem-wide features like group invites and automated consent rules
+- Maintains privacy while improving user experience
+
+### 2. Network Enforcement
+
+Moving from client-side filtering to network-level enforcement:
+
+- Network validates consent before delivering messages
+- Reduces network load from unwanted messages
+- Provides stronger spam prevention
+- Maintains end-to-end privacy
+
+More details about network enforcement will be shared as development progresses.
+
+## Frequently Asked Questions
+
+1. **If I block someone, will they know?**
+
+    No, blocked users aren't notified of their status. From their perspective, it appears the same as if you haven't responded to their message.
+
+2. **Can I revoke consent after granting it?**
+
+   Yes, you can always privately revoke consent. The other party won't be notified, but they'll no longer be able to message you.
```

Comment:
Suggesting a slight tweak here - I think the other party can still message you, but you won't see their messages. 

```suggestion
   Yes, you can always privately revoke consent. The other party won't be notified, but you'll no longer see their messages.
```